src/tests/unit/tests/common/message-creators/action-message-dispatcher.test.ts

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { It, Mock, Times } from 'typemoq';
+import { Message } from '../../../../../common/message';
+import { ActionMessageDispatcher } from '../../../../../common/message-creators/action-message-dispatcher';
+import { Messages } from '../../../../../common/messages';
+import { BaseTelemetryData, TelemetryEventSource } from '../../../../../common/telemetry-events';
+
+describe('ActionMessageDispatcher', () => {
+    const postMessageMock = Mock.ofInstance((message: Message) => {});
+
+    const tabIds = [-1, null];
```

Comment:
It feels off to test *only* edge cases and no realistic ids

Reply:
Separate this cases to remove the `if` block.